src/protocols/bdx/BdxTransferSession.cpp

```diff
@@ -9,15 +9,15 @@
 #include <protocols/Protocols.h>
 #include <protocols/bdx/BdxMessages.h>
 #include <protocols/secure_channel/Constants.h>
+#include <protocols/secure_channel/StatusReport.h>
 #include <support/BufferReader.h>
 #include <support/CodeUtils.h>
 #include <support/ReturnMacros.h>
 #include <system/SystemPacketBuffer.h>
 #include <transport/SecureSessionMgr.h>
 
 namespace {
-constexpr uint8_t kBdxVersion         = 0;         ///< The version of this implementation of the BDX spec
-constexpr size_t kStatusReportMinSize = 2 + 4 + 2; ///< 16 bits for GeneralCode, 32 bits for ProtocolId, 16 bits for ProtocolCode
+constexpr uint8_t kBdxVersion = 0; ///< The version of this implementation of the BDX spec
```

Comment:
nit: should versioning start at 1?

or is this a spec-related versioning number?

Reply:
This is spec-related, and I believe @tcarmelveilleux agreed it should start at 0.